src/views/Auth/components/AuthzManager.vue

```diff
@@ -260,7 +260,7 @@ import {
   updateBuiltInDatabaseData,
   updateAllBuiltInDatabaseData,
 } from '@/api/auth'
-import _ from 'lodash'
+import _, { cloneDeep } from 'lodash'
```

Comment:
Keep styles. If any place uses the `_.xxx` function, only replace to use the `xxx`.